scripts/perf.js

```diff
@@ -0,0 +1,87 @@
+const puppeteer = require('puppeteer');
+const { performance } = require('perf_hooks');
+
+const SERVER = 'localhost';
+const PORT = 5000;
+const APP = 'test/perf/debug';
+
+async function createBrowser() {
+  const browser = await puppeteer.launch();
+
+  return {
+    openPage: async url => {
+      const page = await browser.newPage();
+      await page.goto(url);
+
+      return {
+        close: async () => page.close(),
```

Comment:
These don't need to be marked as `async`. `page.close()` already returns a promise. Might be some eslint-plugin already flags this.